libopenarc/Makefile.am

```diff
@@ -8,7 +8,7 @@ endif
 LDADD = ./libopenarc.la
 
 lib_LTLIBRARIES = libopenarc.la
-libopenarc_la_SOURCES = base64.c arc.c arc.h arc-canon.c arc-canon.h arc-dns.c arc-dns.h arc-internal.h arc-keys.c arc-keys.h arc-tables.c arc-tables.h arc-types.h arc-util.c arc-util.h
+libopenarc_la_SOURCES = base64.c base64.h arc.c arc.h arc-canon.c arc-canon.h arc-dns.c arc-dns.h arc-internal.h arc-keys.c arc-keys.h arc-tables.c arc-tables.h arc-types.h arc-util.c arc-util.h
```

Comment:
However, the other change (from "simple" to "sample") is wrong because the former is the one generated by configure.  The right fix is to add both to openarc/Makefile.am so they get packaged.